dlgr/griduniverse/config.txt

```diff
@@ -2,8 +2,8 @@
 mode = debug
 auto_recruit = true
 network = FullyConnected
-max_participants = 3
-time_per_round = 30
+max_participants = 1
+time_per_round = 3000
```

Comment:
Do we want this change in the PR?

Reply:
No. I should get in the habit of overriding config variables in some other way.